modules/site-health/audit-full-page-cache/load.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Module Name: Audit Full Page Cache
+ * Description: Adds a Full page Cache checker in Site Health checks.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Require helper functions.
+ */
+require_once __DIR__ . '/helper.php';
+
+/**
+ * Adds tests to site health.
+ *
+ * @since 1.0.0
+ *
+ * @param array $tests Site Health Tests.
+ * @return array
+ */
+function perflab_afpc_add_full_page_cache_test( $tests ) {
+	$tests['async']['perflab_page_cache'] = array(
+		'label'             => __( 'Page caching', 'performance-lab' ),
+		'test'              => rest_url( 'performance-lab/v1/tests/page-cache' ),
+		'has_rest'          => true,
+		'async_direct_test' => 'perflab_afpc_page_cache_test',
+	);
+	return $tests;
+}
+add_filter( 'site_status_tests', 'perflab_afpc_add_full_page_cache_test' );
+
+/**
+ * Callback for perflab_page_cache test.
+ *
+ * @since 1.0.0
+ *
+ * @return array
+ */
+function perflab_afpc_page_cache_test() {
+	$description  = '<p>' . __( 'WordPress performs at its best when page caching is enabled. This is because the additional optimizations performed require additional server processing time, and page caching ensures that responses are served quickly.', 'performance-lab' ) . '</p>';
+	$description .= '<p>' . __( 'Page caching is detected by looking for an active page caching plugin as well as making three requests to the homepage and looking for one or more of the following HTTP client caching response headers:', 'performance-lab' )
+					. ' <code>' . implode( '</code>, <code>', array_keys( perflab_afpc_get_page_cache_headers() ) ) . '.</code>';
+
+	$result = array(
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'green',
+		),
+		'description' => wp_kses_post( $description ),
+		'test'        => 'perflab_page_cache',
+		'status'      => 'good',
+		'label'       => '',
+		'actions'     => sprintf(
+			'<p><a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+			esc_url( 'https://wordpress.org/support/article/optimization/#Caching' ),
+			__( 'Learn more about page caching', 'performance-lab' ),
+			/* translators: The accessibility text. */
+			__( '(opens in a new tab)', 'performance-lab' )
+		),
+	);
+
+	$page_cache_detail = perflab_afpc_get_page_cache_detail();
+
+	if ( is_wp_error( $page_cache_detail ) ) {
+		return perflab_afpc_page_cache_unable_detect_cache_test( $result, $page_cache_detail );
```

Comment:
Can you remove this function and put its code right here instead? Breaking this bit out into a new function is unnecessarily complicated since it touches the same `$result` variable, so having that within the same function where its initial values are defined makes it easier to follow.

Also this function isn't used anywhere else, so it shouldn't be necessary.